src/library-authoring/common/context.tsx

```diff
@@ -18,7 +18,8 @@ export interface LibraryContextData {
   isCreateCollectionModalOpen: boolean;
   openCreateCollectionModal: () => void;
   closeCreateCollectionModal: () => void;
-  openCollectionInfoSidebar: () => void;
+  openCollectionInfoSidebar: (collectionId: string) => void
```

Comment:
Nit: Missing semicolon

```suggestion
  openCollectionInfoSidebar: (collectionId: string) => void;
```